index.html

```diff
@@ -307,6 +308,14 @@ <h3>
       </li>
       <li>If |requests| is empty, [=exception/throw=] a {{TypeError}}.
       </li>
+      <li>
+        [=List/For each=] |request| of |requests|:
+        <ol>
+          <li>
+            [=serialize a JavaScript value to a JSON string|Serialize=] |request| to a JSON string.
```

Comment:
Note that "serialize a JavaScript value to a JSON string" can throw, so that needs to be dealt with here. 

```suggestion
            [=serialize a JavaScript value to a JSON string|Serialize=] |request| to a JSON string. [=exception/throw|Re-throw=] any [=exception=].
```

For example, a  cyclic structures:

```
const recursiveObj = {};
recursiveObj.self = recursiveObj;
```